src/_posts/databases/mysql/2000-01-01-backing-up.md

```diff
@@ -0,0 +1,273 @@
+
+We use [On-Demand](#understanding-on-demand-backups) backups to create the
+automated ones. [Manual backups](#understanding-manual-backups) are also
+available for these plans.
+
+
+## Understanding On-Demand Backups
+
+On-Demand backups are done on a daily basis. They consist in dumping your
+database in an archive that we keep during a certain amount of time.
+
+For Business plans, the backup is done on the secondary node to avoid any
+impact on your primary node.
+
+### Retention Policy for On-Demand Backups
+
+We keep a limited amount of backups depending on your database plan:
+- A daily backup is retained for the last 7 days, which means that we will keep
+  a maximum of 7 daily backups (one for each of the last 7 days).
+- A weekly backup means that only one backup is saved over a 7 days period.
+- A monthly backup means that only one backup is saved over the course of a
+  month.
+
+<div class="overflow-horizontal-content" markdown="1">
+| Plan     | Weekly Backups Retained | Monthly Backups Retained |
+| -------- + ----------------------- + ------------------------ |
+| Sandbox  | N/A                     | N/A                      |
+| Starter  | 4 weeks                 | 0 month                  |
+| Business | 8 weeks                 | 12 months                |
+ {: .table }
+ </div>
+
+### Configuring On-Demand Backups
+
+By default, On-Demand backups are done around 1:00 AM Central European Time
+(CET or UTC+0100). This time can be modified.
+
+{% note %}
+The scheduled time is not strongly enforced: it might get delayed depending on
+the workload on our infrastructure.
+{% endnote %}
+
+#### Using the Database Dashboard
+
+1. From your web browser, open your [database dashboard]({% post_url databases/mysql/2000-01-01-getting-started %}#accessing-the-scalingo-for-mysql-dashboard)
+2. Click the **Backups** tab
+3. Locate the **Backup Settings** block
+4. Make sure the **Periodic backups** toggle is set to `enabled`
+5. Chose the time and select the timezone
+6. Click the **Set** button to validate
+
+#### Using the Command Line
+
+1. Make sure you have correctly [setup the Scalingo command line tool]({% post_url platform/cli/2000-01-01-start %})
+2. From the command line, list the addons attached to your application:
+   ```bash
+   scalingo --app my-app addons
+   ```
+   The output should look like this:
+   ```text
+   +-------+-----------------------------------------+-------------------+---------+
+   | ADDON |                   ID                    |       PLAN        | STATUS  |
+   +-------+-----------------------------------------+-------------------+---------+
+   | MySQL | ad-871546ad-943a-4929-9770-ec7c839d65f5 | mysql-starter-512 | running |
+   ...
+   ```
+3. Locate the `ID` corresponding to the addon for which you want to configure
+   the backup schedule
+4. Configure the time of backup:
+   - By setting an hour:
+     ```bash
+     scalingo --app my-app addon <addon_ID> backups-config --schedule-at 3
```

Comment:
suggestion: this is the same here. No need to retrieve the UUID. You can use the database type. And I'm pretty sure it's the case for all commands

Reply:
:exploding_head: I didn't know that at all, thanks for the tip.
Fixed in https://github.com/Scalingo/documentation/pull/2501/commits/28b2dc598554cc4a14cf1e6e7fe1f945a13942ec